client/components/tinymce/plugins/simple-payments/dialog/form.jsx

```diff
@@ -30,6 +30,15 @@ export const getProductFormValues = state => getFormValues( REDUX_FORM_NAME )( s
 export const isProductFormValid = state => isValid( REDUX_FORM_NAME )( state );
 export const isProductFormDirty = state => isDirty( REDUX_FORM_NAME )( state );
 
+// based on https://stackoverflow.com/a/10454560/59752
+function decimalPlaces( number ) {
+	const match = ( '' + number ).match( /(?:\.(\d+))?(?:[eE]([+-]?\d+))?$/ );
```

Comment:
We could ignore the scientific notation here and simplify the function a lot. But just copying the Stack Overflow answer is perfectly OK, too.